apps/barista-design-system/src/pages/single-page/components/toc/toc.ts

```diff
@@ -22,14 +22,14 @@ import {
   OnDestroy,
   NgZone,
   Inject,
+  Input,
+  QueryList,
 } from '@angular/core';
-import {
-  BaTocService,
-  BaTocItem,
-} from '../../../../shared/services/toc.service';
+import { BaTocService } from '../../../../shared/services/toc.service';
 import { Subscription } from 'rxjs';
 import { Platform } from '@angular/cdk/platform';
 import { DOCUMENT } from '@angular/common';
+import { TOC } from '@dynatrace/shared/barista-definitions';
```

Comment:
When importing it from shared, please do not use abbreviations. `TableOfContents` is fine